app/views/request/_sidebar.html.erb

```diff
@@ -1,49 +1,66 @@
 <div id="right_column" class="sidebar right_column" role="complementary">
-  <div class="new-request-cta">
+  <div class="sidebar__section new-request-cta">
     <%= render :partial => "general/new_request" %>
   </div>
 
+  <% if show_pro_upsell?(@user) %>
+    <div class="sidebar__section pro-upsell">
+      <% pro_link = link_to account_request_index_path do %>
+        <%= AlaveteliConfiguration.pro_site_name -%>
+      <% end %>
+
+      <% pro_site_name_link = content_tag(:strong, pro_link) %>
```

Comment:
Extract to helper? Avoids variable assignment in the views and so its testable too?

Reply:
I couldn't decide about this either. I was thinking the whole thing should be a helper; something like:

```ruby
<= pro_upsell if show_pro_upsell?(@user) %>
```

But, if there's success here, I'd like to also add it to one of the slots under the search bar in `/select_authority`, which would be a good time to refactor common bits? I didn't really want to delay this while I think about that.

I can't really think of a good name for a helper that does just the link at the moment. Any thoughts? It seems like it needs to be something like `strong_pro_site_name_link_to_account_request_index` to be explicit about what its doing, but that's rather unwieldy!

Reply:
I guess there could be a helper for just the text part:

```erb
<% if show_pro_upsell?(@user) %>
  <div class="sidebar__section pro-upsell">
    <%= pro_upsell %>
  </div>
<% end %>
```

Reply:
Fixed in a91111a7d.